doc/src/man/litani-get-jobs.scdoc

```diff
@@ -0,0 +1,45 @@
+litani-get-jobs(1) "" "Litani Build System"
+
+; Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+; Licensed under the Apache License, Version 2.0 (the "License").
+; You may not use this file except in compliance with the License.
+; A copy of the License is located at
+
+;     http://www.apache.org/licenses/LICENSE-2.0
+
+; or in the "license" file accompanying this file. This file is distributed
+; on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+; express or implied. See the License for the specific language governing
+; permissions and limitations under the License.
+
+
+# NAME
+
+litani get-jobs - Print a list of jobs to be run
+
+
+# DESCRIPTION
+
+This program prints the list of jobs that have been added so far in JSON
+format. Each job in the list will follow the same schema as the elements of the
+*["jobs"]["wrapper_arguments"]* key in the litani _run.json_ file; see
+*litani-run.json(5)* for the schema.
+
+
+# MOTIVATION & BEST PRACTICES
```

Comment:
A few things worth mentioning IMHO:
* This command may be run at any point after running `litani init`---either before calling `litani run-build`, during the run, or after run-build has terminated.
* Multiprocess safety should be a whole other section. This command is not safe to run concurrently with `set-jobs`, because `set-jobs` might delete a JSON file while this command reads it. This command is safe to run concurrently with `add-jobs` and `run-build` because new job files get written atomically (and `run-build`/`exec` write files atomically, but don't mention `exec`), so this command will either read the whole file or miss it completely.
* Compared to `dump-run`, this command will always return immediately if it's successful; `dump-run` has to lock the build while it runs. So if you just need the list of jobs and not other metadata of the run, this command might be better for you. On the other hand: the run contains the Litani version number, which you might want to use for checking that a certain key is part of the schema, so in general it can help you write more robust client code.
* Compared to `transform-jobs`, this command is obviously useful if you just want to read the jobs and not write them... but a usage difference is that clients of `transform-jobs` have to access the stdin and stdout of a subprocess, while this command can use input/output files and can be used in a Unix pipeline, which may be more natural.
* Can I suggest a short (for now) EXAMPLES section, maybe even just one example showing a pipeline? I haven't tested this but
```
# Pipeline
litani get-jobs | jq '[.[] | (.wrapper_arguments.command = "echo " + .wrapper_arguments.command)]' | litani set-jobs

# Sequence
litani get-jobs --out-file /tmp/original.json
jq '[.[] | (.wrapper_arguments.command = "echo " + .wrapper_arguments.command)]' < /tmp/original.json > /tmp/transformed.json
litani set-jobs --from-file /tmp/transformed.json
```
will prepend `echo ` to each command.